### PR TITLE
init commit for ctuguina

### PR DESCRIPTION
**Pull request by ctuguinay**
* add caesar tuguinay avatar
* add caesar tuguinay index
* add caesar tuguinay CV

### CODE REVIEW

**Comment by ctuguinay** [comment]
@leewujung

**Comment by leewujung** [comment]
@ctuguinay looks good! I’ll merge this now.

**Comment by ctuguinay** [comment]
@leewujung I just pushed a really small change.

File: content/authors/ctuguina/_index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+# Enter email to display Gravatar (if Gravatar enabled in Config)
+# email: ""
+
+# Organizational groups that you belong to (for People widget)
+#   Set this to `[]` or comment out if you are not using People widget.
+user_groups:
+- Researchers
+---
+
+I am a student at the University of Washington majoring in Mathematics and minoring in Data Science. I am expected to graduate the Spring of 2023 and am planning on pursuing a graduate education in Statistics.
```
Comment (leewujung) [inline content/authors/ctuguina/_index.md:69]:
```suggestion
I am a student at the University of Washington majoring in Mathematics and minoring in Data Science. I am expected to graduate in the Spring of 2023 and am planning on pursuing graduate education in Statistics.
```

File: content/authors/ctuguina/_index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+My applied math interests are in signal processing, specifically in the area of acoustics. My math interests lie mostly in Harmonic Analysis. My hobbies include running, swimming, and (very inconsistently) playing the piano.
+
+At Echospace, I am working on the Echoregions software package and the UBNA Bat Passive Acoustic Monitoring project.
```
Comment (leewujung) [inline content/authors/ctuguina/_index.md:73]: add link to Echoregions repo?

**Pull request merged**